From f8019bd7de491e487fb88272304b54eb9a5d56f6 Mon Sep 17 00:00:00 2001
From: Murala S Swamy <ssmurala@habana.ai>
Date: Thu, 12 Sep 2024 12:03:45 +0300
Subject: [PATCH] [FSW-6580] fs1: set lfs done after boot info set

With this patch, added support to set lfs done
progress bit once boot rom completes writing
boot info using lfs file.

Change-Id: I3654baab3760edd96390f124e6d626dac00abb8f
---

diff --git a/arch/fs1/arc/dev_boot_info.c b/arch/fs1/arc/dev_boot_info.c
index f91c861..56ea156 100644
--- a/arch/fs1/arc/dev_boot_info.c
+++ b/arch/fs1/arc/dev_boot_info.c
@@ -47,5 +47,8 @@
 		boot_rom_status_update(PROGRESS_BOOT_INFO_IMG_DESC_INFO_SET_FINISH);
 	}
 
+	/* Flag which is indication for other modules to know Boot ROM done with LFS */
+	boot_rom_status_update(PROGRESS_BOOT_INFO_LFS_FINISHED);
+
 	return;
 }
